### PR TITLE
Adding Heatmap to pyplot: plt.heat

### PR DESCRIPTION
**Pull request by rmenegaux**

### CODE REVIEW

File: bqplot/pyplot.py [hunk 1]
```diff
@@ -854,6 +854,25 @@ def geo(map_data, **kwargs):
     return _draw_mark(Map, **kwargs)
 
 
+def heat(color, **kwargs):
```
Comment (dmadeka) [inline bqplot/pyplot.py]: heat_map? What does mpl do?

File: bqplot/marks.py [hunk 2]
```diff
@@ -42,7 +42,7 @@
     )
 from traittypes import Array
 
-from .scales import Scale, OrdinalScale
+from .scales import Scale, OrdinalScale, LinearScale
```
Comment (dmadeka) [inline bqplot/marks.py]: Is Ordinal still used?

**Comment by dmadeka** [comment]
@rmenegaux  Id like a syntax like https://discuss.analyticsvidhya.com/t/how-to-plot-heat-map-in-python/1922/2

``` python
plt.pcolor or plt.map(data, cmap='Reds')
```

@ssunkara1 what do you think?

**Comment by ssunkara1** [comment]
@dmadeka @rmenegaux How about `plt.heatmap` ? I suggest that because that seems the most intuitive one to me.

File: bqplot/marks.py (hunk 2 above)
Comment (rmenegaux) [inline bqplot/marks.py]: @dmadeka Yes

**Comment by rmenegaux** [comment]
Okay going with heatmap then

**Pull request merged**